service/trino-verifier/src/main/java/io/trino/verifier/JsonEventClient.java

```diff
@@ -52,7 +53,7 @@ public <T> void postEvent(T event)
             ByteArrayOutputStream buffer = new ByteArrayOutputStream();
             JsonGenerator generator = factory.createGenerator(buffer, JsonEncoding.UTF8);
             serializer.serialize(event, generator);
-            out.println(buffer.toString().trim());
+            out.println(buffer.toString(UTF_8).trim());
```

Comment:
what is `out`? sounds like default charset can be appropriate, but idk

Reply:
```java
this.out = new PrintStream(config.getEventLogFile());
```

IOW, this is a log file.